buildtest/buildsystem/base.py

```diff
@@ -319,11 +336,7 @@ def get_environment(self):
 
         env = []
         pairs = self.recipe.get("env")
```

Comment:
If you do self.recipe.get("env", []) you don't need to do `if pairs` but can just loop through them still (an empty list).